src/client/unittests/pytest/runner.ts

```diff
@@ -78,7 +78,14 @@ export function runTest(rootDirectory: string, tests: Tests, args: string[], tes
             });
 
             launchDef.promise.then(() => {
-                return vscode.commands.executeCommand('vscode.startDebug', {
+                if (!Array.isArray(vscode.workspace.workspaceFolders) || vscode.workspace.workspaceFolders.length === 0) {
```

Comment:
Is this code any different from the code in the nosetest runner (same for unittest)? If not then is it worth factoring it out to a helper function?